### PR TITLE
Make feature tests running on GPU and Default Raw

### PR DESCRIPTION
**Pull request by bdattoma**
Adapting `ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_tgis.robot` suite to run with GPU and RawDeployment (as cluster default option, no annotation)

It adds also the tag ServerlessOnly for some KServe feature tests like Canary and Autoscaling which we'll test later outside this cycle

### CODE REVIEW

**Comment by bdattoma** [comment]
PR validation:` rhods-ci-pr-test/2593`

File: ods_ci/tests/Resources/CLI/ModelServing/llm.resource [hunk 1]
```diff
@@ -723,3 +742,21 @@ Download Prompts Weights In PVC
     Wait For Pods To Be Ready    label_selector=name=${model_name}-download-prompts    namespace=${namespace}
     Wait For Pods To Succeed    label_selector=name=${model_name}-download-prompts    namespace=${namespace}
     ...    timeout=${download_timeout}
+
+Get KServe Default Deployment Mode From DSC
+    [Arguments]    ${dsc_name}=${DSC_NAME}
+    ${rc}    ${mode}=    Run And Return Rc And Output
+    ...    oc get dsc ${DSC_NAME} -ojsonpath='{.spec.components.kserve.defaultDeploymentMode}'
+    Run Keyword And Continue On Failure    Should Be Equal As Integers    ${rc}    ${0}    msg=${mode}
+    IF    "${mode}" == "${EMPTY}"
```
Comment (tarukumar) [inline ods_ci/tests/Resources/CLI/ModelServing/llm.resource:751]: i think you need to add or condition also here to check if the deployment mode is set to Serverless or not

Comment (bdattoma) [reply ods_ci/tests/Resources/CLI/ModelServing/llm.resource:751]: why?

**Review by jgarciao — APPROVED** [review]

**Review by tarukumar — APPROVED** [review]

File: ods_ci/tests/Resources/CLI/ModelServing/llm.resource (hunk 1 above)
Comment (tarukumar) [reply ods_ci/tests/Resources/CLI/ModelServing/llm.resource:751]: explain in the chat :)

**Pull request merged**